lib/functions.php

```diff
@@ -127,3 +127,24 @@ function ppi_delete_user_working_designs() {
   global $wpdb;
   $wpdb->delete($wpdb->usermeta, array('meta_key' => 'pp_working_design_id'));
 }
+
+
+/**
+ * Extract registerable domain from string input
+ *
+ * @param string $url
+ * @return string|false
+ */
+function ppi_extract_domain($url) {
```

Comment:
this function I just took wholesale from https://github.com/netrivet/common/blob/master/classes/NrUtil.php#L333 which is how we get registrable domains in P5 and P6 for retrieving nameservers.